src/test/java/com/paypal/base/BaseTest.java

```diff
@@ -52,19 +53,10 @@ public void setUp() {
     }
 
     @AfterClass
-    public void
-    tearDown() {
+    public void tearDown() {
         driver.quit();
     }
 
-    public void pauseBrowser(int seconds) {
-        try {
-            Thread.sleep(seconds * 1000L);
-        } catch (InterruptedException e) {
-            throw new RuntimeException(e);
-        }
-    }
-
     public Transaction getTransaction() {
         CreditCard card = new CreditCard("John", "376680816376961",
```

Comment:
Why username, credit card number are visible here

Reply:
Thanks for pointing this out. I will move them into properties file.